src/widget/wtrackmenu.h

```diff
@@ -34,7 +34,7 @@ class WTrackMenu : public QMenu {
         HideUnhidePurge = 1 << 9,
         FileBrowser = 1 << 10,
         Properties = 1 << 11,
-        TrackModelFilters = LoadTo | Remove | HideUnhidePurge
+        TrackModelFilters = Remove | HideUnhidePurge
```

Comment:
"Filter" isn't a very specific name for this enum. How about renaming it to "MenuFeatures"? Also rename "TrackModelFilters" to "TrackModelFeatures".

Reply:
Thinking about it agian, "Menu" isn't really necessary in that name because it's already in the WTrackMenu namespace. Simply "Features" instead of "Filter" works.

Reply:
Cool. "Options" is another possible name. Should we use it or stick to "Features"?

Reply:
"Options" isn't bad, but I think "Features" fits better in this context.